<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fotograma</class>
 <widget class="QDialog" name="Fotograma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fotograf√≠a caballo</string>
  </property>
  <widget class="QLabel" name="label_fondo_foto">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>460</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>520</width>
     <height>460</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>520</width>
     <height>460</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(91, 57, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_foto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>441</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_numero_vuelta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 9pt &quot;AcadEref&quot;;
color: rgb(0,0,0);
background-color: rgb(255, 159, 28);</string>
   </property>
   <property name="text">
    <string>Comienzo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_tiempo">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>26</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 87 26pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>00:00.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
